apptainer/moose-dev.def

```diff
@@ -46,7 +43,7 @@ From: {{ APPTAINER_FROM }}
 
 %environment
     export PATH={{ MOOSE_MINIFORGE }}/bin:/opt/code-server/bin:$PATH
-    source activate {{ MOOSE_MINIFORGE }}/envs/{{ MOOSE_CONDA_ENV_NAME }}
+    source activate {{ MOOSE_MINIFORGE }}
```

Comment:
@loganharbour This is my proposed fix. e.g. Still need to activate the base environment.